assets/js/components/indicators.js

```diff
@@ -296,6 +302,9 @@ export class IndicatorSection {
       button.click(() => {
         this.chart.highlightCol(comparison.municipality.code);
         ga('send', 'event', 'chart-compare-highlight', `${this.key} ${comparison.municipality.code}`);
+        gtag('event', 'chart_compare_highlight', {
+          chart: `${this.key} ${comparison.municipality.code}`,
```

Comment:
let's put this in `label` instead of `action`

Reply:
Changed to label